src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,53 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.0
+
+### New features
+
+* Added a new `subctl uninstall` command that removes all Submariner components and dataplane artifacts, such as IP table rules, routes from
```

Comment:
Are we trying to say "IP table rules" and "IP table routes"? If so, I think it should be:

```suggestion
* Added a new `subctl uninstall` command that removes all Submariner components and dataplane artifacts, such as IP table rules and routes, from
```

Also I guess it should be "iptables":

https://submariner.io/development/website/style-guide/#submarinerio-word-list

Reply:
Yes, we should use iptables.